turbo.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": ["src/**/*.ts"],
+      "outputs": ["dist/**"]
+    },
+    "typecheck": {
+      "inputs": ["src/**/*.ts"]
+    },
+    "lint:check": {
+      "inputs": ["src/**/*.ts"]
+    },
+    "format:check": {}
```

Comment:
why this doesn't have inputs?

Reply:
Because the input for prettier is "all files"